src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -377,10 +377,11 @@ public static String nullSafeGetString(final JSONObject data, final String key)
    * @return a JSONArray containing a specific subset of tags retrieved from a builds metadata.
    */
   public static JSONArray assembleTags(final JSONObject builddata, final HashMap<String,String> extra) {
+    DatadogBuildListener.DescriptorImpl descriptor = DatadogUtilities.getDatadogDescriptor();
     JSONArray tags = new JSONArray();
 
     tags.add("job:" + builddata.get("job"));
-    if ( (builddata.get("node") != null) && DatadogUtilities.getDatadogDescriptor().getTagNode() ) {
+    if ( (builddata.get("node") != null) && descriptor.getTagNode() != null && descriptor.getTagNode() ) {
```

Comment:
This was throwing a NPE upon completion of every build if you were running Jenkins with the Datadog plugin installed but not configured.

The problem was the `tagNode` variable (in DatadogBuildListener) is null, but the `getTagNode` method tries to coerce it to a Boolean and fails.  Another possible way to fix this could be tomake `getTagNode` detect this case and just return `false` instead.  I'm happy to make this change if you feel that it's better.